packages/input/__tests__/input.spec.ts

```diff
@@ -273,6 +273,18 @@ describe('Input.vue', () => {
       expect(input.selectionStart).toEqual(0)
       expect(input.selectionEnd).toEqual(testContent.length)
     })
+    test('method:resizeTextarea', async () => {
+      const testContent = 'TEXT:resizeTextarea'
+      const wrapper = _mount({
+        template: `<el-input  ref="textarea"  type="textarea" v-model="text" />`,
+        data() {
+          return {
+            text: testContent,
+          }
+        },
+      })
+      wrapper.vm.$refs.textarea.resizeTextarea()
```

Comment:
Could you make it measurable? Because I can't see what you were trying to test. For instance, test some value that changes inside `resizeTextarea`

Reply:
i got it